HW_3/task_7.py

```diff
@@ -0,0 +1,24 @@
+''' Задание №7
+Пользователь вводит строку текста.
+Подсчитайте сколько раз встречается каждая буква в строке без использования метода count и с ним.
+Результат сохраните в словаре, где ключ —символ, а значение — частота встречи символа в строке.
+Обратите внимание на порядок ключей. Объясните почему они совпадают или не совпадают в ваших решениях.
+'''
+
+# text = input("Введите строку текста: ")
+
+# frequency = {}
+# for char in text:
+#     frequency[char] = frequency.get(char, 0) + 1
+
+# print("Результат (без использования метода count):")
+# print(frequency)
+
+text = input("Введите строку текста: ")
+
+frequency = {}
+for char in text:
```

Comment:
Тут можно добавить использование множеств.
```suggestion
for char in set(text):
```